### DISCUSSION TITLE
quick filter in editor.ts

### DISCUSSION THREAD

**Discussion by hosnabigdeli** [serenity-is/Serenity#5882] [Help]
hi 
how can i add quick filter in editor.ts.

**Comment by teafmartins** [comment]
@MHH56, did you get it? I'm looking for that too.

**Reply by hosnabigdeli** [reply]
Yes solved with this code:


`@Serenity.Decorators.filterable()


protected enableFiltering() {
	return true;
}

protected onViewSubmit() {

	this.view.beginUpdate();

	let filter = function filter(item, args) {
		var linha = (item as TduLinhasConfiguracaoArticuladoRow);

		
		if (item["ReviewResult"] == "1")
			return true;

		return false;
	}

	this.view.setFilter(filter);

	this.view.endUpdate();

	return true;
}`
